src/fidesops/ops/models/privacy_request.py

```diff
@@ -685,7 +686,6 @@ class ProvidedIdentity(Base):  # pylint: disable=R0904
     privacy_request_id = Column(
         String,
         ForeignKey(PrivacyRequest.id_field_path),
-        nullable=False,
```

Comment:
I see this is specified in the ticket, but where are privacy requests becoming non-nullable?

Reply:
I was wondering this also. The [other part](https://github.com/ethyca/fidesops/issues/1161) of the ticket shows receiving the data as the following so I think we are going to create an identity and consent with no privacy request here.

```json
{
  "identity": "test@example.com",  # should this be hashed?
  "identity_type": "email",
  "regulation": "gdpr",
  "data_use": "this can validated against Fideslang until we unify Fides",  # we won't support custom data uses
  "opt_in": "true"
}
```

The other thing I was questioning here is if we really need a new table for this or if we should just add the `opt_in` to the `ProvidedIdentity`. Since it's a 1 to 1 relationship it seems like they could just go together.

Reply:
ah I see. Maybe this could go on the ProvidedIdentity if we're sure it's going to stay a 1:1, I'd check with someone else who's more informed about this feature-

Reply:
discussed in slack, we will be creating ProvidedIdentities that are not related to PrivacyRequests.  and there can be multiple data uses for a single identity.